lib/parzival_web/live/app/vault/order_live/index.ex

```diff
@@ -18,9 +18,25 @@ defmodule ParzivalWeb.App.OrderLive.Index do
      socket
      |> assign(:current_page, :vault)
      |> assign(:params, params)
+     |> assign(list_products(params))
      |> assign(list_orders(params, socket))}
   end
 
+  defp list_products(params) do
+    case Store.list_products(params) do
+      {:ok, {products, meta}} ->
+        products =
+          products
+          |> Enum.filter(fn product -> product.stock > 0 end)
+          |> Enum.take(3)
```

Comment:
Why do you need to take only the first 3?

Try to filter products out of stock by filtering directly on the query. However, I think they should be gray and not say "sold out" or something.